api-put-object-fan-out.go

```
@@ -69,6 +69,7 @@ type PutObjectFanOutResponse struct {
 // stream multiple objects are written, defined via a list of PutObjectFanOutRequests. Each entry
 // in PutObjectFanOutRequest carries an object keyname and its relevant metadata if any. `Key` is
 // mandatory, rest of the other options in PutObjectFanOutRequest are optional.
+// Deprecated: This functionality will be deprecated in future releases, please use PutObject instead.
```

golangci-lint (2.12.2)

[error] deprecatedComment: `Deprecated: ` notices should be in a dedicated paragraph, separated from the rest

(gocritic)